src/rgw/driver/sfs/object.cc

```diff
@@ -282,26 +287,98 @@ void SFSObject::gen_rand_obj_instance_name() {
   state.obj.key.set_instance(buf);
 }
 
-int SFSObject::get_obj_attrs(
-  optional_yield y,
-  const DoutPrefixProvider *dpp,
-  rgw_obj *target_obj
-) {
-  lsfs_dout(dpp, 10) << " TODO" << dendl;
+/*
+  This should be intended as a mere fetch of object's attributes.
+  The caller will likely call get_attrs() after get_obj_attrs().
```

Comment:
get_attr?

Reply:
can you please elaborate more precisely your question?

Reply:
Is get_attrs a typo? There is no such function in object.cc. There is however a get_attr